### PR TITLE
Fix fromUnixTimestamp64 functions

### LINKED ISSUE #14648: fromUnixTimestamp64 functions expect Int64 and not UInt64
SELECT toYYYYMMDD(fromUnixTimestamp64Micro(1599672041963889))

Received exception from server (version 20.9.1):
Code: 43. DB::Exception: Received from localhost:9000. 
DB::Exception: Illegal type of argument #0 'value' of function fromUnixTimestamp64Micro, expected Int64, got UInt64.

### PR DESCRIPTION
**Pull request by zvonand**
Changelog category (leave one):
- Improvement

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Improvement for `fromUnixTimestamp64` family functions.. They now accept any integer value that can be converted to `Int64`. This closes: #14648.

### CODE REVIEW

**Comment by zvonand** [comment]
CC @Enmk

**Comment by den-crane** [comment]
There is no test?  `SELECT fromUnixTimestamp64Micro(1599672041963889);` ?

**Comment by zvonand** [comment]
> There is no test? `SELECT fromUnixTimestamp64Micro(1599672041963889);` ?

There is a test, but it was crossed out for some different reason a long ago. Will review it a bit later.

**Review by alexey-milovidov — COMMENTED** [review]
Mostly LGTM

**Comment by alexey-milovidov** [comment]
Style check is failed - merge / rebase with master to get it fixed.

**Comment by alexey-milovidov** [comment]
Why don't add support for floating point types?
It should be expected for DataTime64.

**Comment by zvonand** [comment]
> Why don't add support for floating point types?
> It should be expected for DataTime64.

I don't think there is a point. `toDateTime64()` supports non-int types. These functions are not taking timestamps in seconds. 
It seems to me that e.g. 1234347.4934 milliseconds value can be quite confusing :)

**Comment by alexey-milovidov** [comment]
But `fromUnixTimestamp64(1642206185.333)` makes sense.

**Comment by alexey-milovidov** [comment]
@zvonand There are a few changes remaining.

**Comment by alexey-milovidov** [comment]
> Fix fromUnixTimestamp64...() functions.

This is not a changelog entry for user.

**Comment by alexey-milovidov** [comment]
@zvonand Only a few changes remaining.

File: tests/queries/0_stateless/01277_fromUnixTimestamp64.sql [hunk 1]
```diff
@@ -42,4 +42,30 @@ SELECT
 	i64,
 	fromUnixTimestamp64Milli(i64, tz),
 	fromUnixTimestamp64Micro(i64, tz),
-	fromUnixTimestamp64Nano(i64, tz) as dt64;
\ No newline at end of file
+	fromUnixTimestamp64Nano(i64, tz) as dt64;
```
[the remark this answers is not in the document]
Comment (zvonand) [reply tests/queries/0_stateless/01277_fromUnixTimestamp64.sql:45]: Test remains quite simple here - no sense in making a new one. Basically, they all test one thing - why separate them?

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**